client/src/components/payment/upgrade-modal.vue

```diff
@@ -50,7 +57,24 @@ import { getBaseUrl } from './payment-utils'
     }
   }
 })
-export default class UpgradeModal extends Mixins(NameRequestMixin, PaymentMixin, PaymentSessionMixin) {
+export default class UpgradeModal extends Mixins(
+  NameRequestMixin,
+  PaymentMixin,
+  PaymentSessionMixin,
+  DisplayedComponentMixin
+) {
+  /**
+   * Optionally display the countdown timer.
+   * This could be turned into a prop for easier configuration.
+   */
+  get displayTimer () {
+    return false
```

Comment:
Is this just debugging code? If not, why does this exist?

Reply:
Ditto timerName below...?

Reply:
That just allows you to optionally display timers on the update, receipt, and re-apply pages should you want to. Lorna and I were discussing whether or not we wanted timers on those, but then she went on holiday.

Reply:
Where is the conditional? As far as I can tell, this always returns false.

How about adding a comment indicating this is for possible future use?